README.md

```diff
@@ -1 +1,100 @@
 # hybro
+
+ReactNative <-> Web invoke/events communication
+
+---
+
+## ReactNative API
+
+### Installation
+
+Install and link [react-native-webview](https://github.com/react-native-community/react-native-webview)
+
+```
+npm i react-native-webview
+react-native link react-native-webview
+```
+
```

Comment:
Add hybro install line

Reply:
I think we miss installation hybro in web app. And this part should be moved from RN part of docs to common.